app/keycloak.js

```diff
@@ -103,6 +105,16 @@ export const keycloakMiddleware = (api) => (storeAPI) => (next) => (action) => {
       break;
     }
     default:
+      if (
+        action?.error?.status === 401 ||
+        action?.error?.originalError?.status === 401 ||
+        action?.error?.status === 403 ||
```

Comment:
I can see how this makes perfect sense for 401.

More of a philosophical question here: will re-login solve a 403?

I suppose won't hurt to try, but if they didn't have the permission before, it seems unlikely that they would gain it after logging in again - unless someone just granted them the permission.

Reply:
There's some amount of historical ambiguity around which status the backend returns in which situation. There have been cases where we should have used one but return the other, so this code treats both the same way.